_partials/rightcol.html.haml

```diff
@@ -2,27 +2,27 @@
   %h2 Getting Started
   %ul.nav.nav-pills.nav-stacked
     %li
-      %a{:href => href="https://www.jboss.org:80/narayana/downloads/latest.html"}
+      %a{:href => href="#{site.base_url}/downloads/index.html"}
         %i.icon-fixed-width.icon-arrow-right
         Download the latest version of Narayana standalone
     %li
-      %a{:href => href="https://ci.jboss.org/hudson/job/WildFly-latest-master/"}
+      %a{:href => href="https://www.wildfly.org/downloads/"}
         %i.icon-fixed-width.icon-arrow-right
-        Or download the latest version of Narayana + WildFly
+        Or download the latest release of WildFly
     %li
-      %a{:href => href="https://www.jboss.org:80/narayana/documentation/latest.html"}
+      %a{:href => href="#{site.base_url}/documentation/index.html"}
         %i.icon-fixed-width.icon-arrow-right
         Read our getting started documentation
     %li
       %a{:href => href="https://github.com/jbosstm/quickstart"}
         %i.icon-fixed-width.icon-arrow-right
         Run our quickstarts
     %li
-      %a{:href => href="https://narayana.io/community/index.html"}
+      %a{:href => href="#{site.base_url}/community/index.html"}
         %i.icon-fixed-width.icon-arrow-right
         Meet the team
     %li
-      %a{:href => href="https://community.jboss.org/wiki/GetInvolvedWithNarayana"}
+      %a{:href => href="https://github.com/jbosstm/narayana/blob/master/CONTRIBUTING.md"}
```

Comment:
@mmusgrov is all the pertinent information from https://developer.jboss.org/docs/DOC-51434 in the CONTRIBUTING.md